paasta_tools/utils.py

```diff
@@ -228,6 +228,11 @@ class PersistentVolume(VolumeWithMode):
     storage_class_name: str
 
 
+class SecretVolume(VolumeWithMode):
```

Comment:
nit: it looks like we don't use the mode and always set `read_only=True`, so this could just inherit from TypedDict